src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/TplEventSource.cs

```diff
@@ -10,13 +10,7 @@ namespace System.Threading.Tasks
     /// <summary>Provides an event source for tracing TPL information.</summary>
     [EventSource(
         Name = "System.Threading.Tasks.TplEventSource",
-        Guid = "2e5dba47-a3d2-4d16-8ee0-6671ffdcd7b5",
-        LocalizationResources =
-#if CORECLR
```

Comment:
This was set up in https://github.com/dotnet/coreclr/pull/22120. I don't think we really want to keep it this way, so I fixed that by adding Message unconditionally.